### PR TITLE
Fix parsing & printing of es6 function syntax inside attribrutes

### PR DESCRIPTION
**Pull request by IwanKaramazow**
Fixes https://github.com/facebook/reason/issues/1937
Fixes https://github.com/facebook/reason/issues/1902

### CODE REVIEW

File: src/reason-parser/reason_lexer.mll [hunk 1]
```diff
@@ -919,6 +919,7 @@ and skip_sharp_bang = parse
 
   let inject_es6_fun = function
     | tok :: acc ->
+        print_endline "INJECTING";
```
Comment (hcarty) [inline src/reason-parser/reason_lexer.mll]: Is this intentional?

Comment (chenglou) [inline src/reason-parser/reason_lexer.mll]: lol?

Comment (IwanKaramazow) [reply src/reason-parser/reason_lexer.mll]: forgot to rebase lol

Comment (IwanKaramazow) [reply src/reason-parser/reason_lexer.mll]: Forgot to remove it 💃

**Comment by jordwalke** [comment]
I think I ended up having a good use case for this one. Glad to see this fix!

**Pull request merged**